source/client/controllers/login-controller.js

```diff
@@ -8,7 +8,8 @@ Space.messaging.Controller.extend(Space.accountsUi, 'LoginController', {
   eventSubscriptions() {
     return [{
       'Space.accountsUi.LoginRequested': this._onLoginRequested,
-      'Space.accountsUi.LogoutRequested': this._onLogoutRequested
+      'Space.accountsUi.LogoutRequested': this._onLogoutRequested,
+      'Space.accountsUi.LoginWithGoogleRequested': this._onLoginWithGoogleRequested,
```

Comment:
Ok then only unify the failure / success events? My concern here is that subscribers need too much detail about the login process otherwise. Imagine you allow login with all different service providers. Then you would need 6 different event handlers although the basic flow (showing errors / login successful) is always the same :wink: